metadata_generation.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
+import os
+import argparse
+
+
+def combine_metadata(
+        metadata_csv:str,
+        chexpert_csv:str,
+        negbio_csv:str,
+        split_csv:str,
+        output_dir:str,
+) -> None:
```

Comment:
Please use 4 inputs rather than 5 i.e.

```
meta_path: str,
findings_path: str,
split_path: str,
save_dir: str,
```
**findings_path** is one of csv files i.e. _chexpert_ or _negbio_